helm/oauth2-proxy/README.md

```diff
@@ -5,6 +5,7 @@
 ## TL;DR;
 
 ```console
+$ helm repo add oauth2-proxy https://oauth2-proxy.github.io/manifests
 $ helm install stable/oauth2-proxy
```

Comment:
Is it still called `stable/oauth2-proxy`? Is that what we want to have it called?

Reply:
It's whatever you "add" the repo as (repo/chart). So in this case it should be
```suggestion
$ helm install oauth2-proxy/oauth2-proxy
```